src/pharmpy/tools/external/nonmem/results.py

```diff
@@ -86,8 +86,8 @@ def _parse_modelfit_results(
     (
         runtime_total,
         log_likelihood,
-        covstatus,
-        covstatus_table_number,
+        parameter_uncertainty_status,
```

Comment:
Since this is parsing the NONMEM results I think we should keep covstatus as being the status of $COV. For other methods there could be separate status if needed.